region/brinstar/kraid.json

```diff
@@ -314,6 +314,15 @@
                 "requires": null,
                 "openEnd": 1
               }
+            ],
+            "canLeaveCharged": [
+              {
+                "usedTiles": 31,
+                "framesRemaining": 0,
+                "shinesparkFrames": 15,
+                "requires": [ {"resetRoomAtNode": [1, 3]} , "Morph" ],
```

Comment:
That must require something to break the bomb block as well, surely? Basically access from 1 to 3. Bit annoying that we're back to duplicating some logic, but at least it's not widespread

Reply:
Indeed, ops.